tap_twitter_ads/__init__.py

```diff
@@ -21,8 +22,26 @@
     'account_ids'
 ]
 
-def do_discover(reports):
+# checking credentials for the discover mode
+def check_credentials(client, account_ids):
+    # check whether tokens are valid or not
+    get_resource('accounts', client, 'accounts')
```

Comment:
It will throw a readable message? Do we need this? It won't be covered as part of line no. 33 if it is throwing a readable message: 
`client.accounts(account_id)`

Reply:
Yes, it is using SDK so it will throw an error message.  We kept one extra API call above to check the token before validating account_ids.